.pre-commit-config.yaml

```diff
@@ -42,6 +47,6 @@ repos:
         args:
           - --max-line-length=120
           - --min-public-methods=0
-          - --good-names=q,f,fp,i,e
-          - --disable=E0401,W1201,W1203,W0603,C0114,C0115,C0116,C0411
+          - --good-names=q,f,fp,i,e,af
```

Comment:
actually, `af` might be better renamed : it's supposed to stand for `automation framework`, maybe we could rename it `automation` ?

Reply:
it is used to store automation config so I renamed it as `automation_config`